python/tests/rtl/test_requests_transport.py

```diff
@@ -0,0 +1,15 @@
+"""Test the requests transport.
+"""
+
+from looker.rtl import transport as tp
+from looker.rtl import requests_transport as rtp
+
+
+def test_request_minimal():
+    """Test basic successful round trip
+    """
+    settings = tp.TransportSettings(base_url='/some/path', api_version='3.1')
+    test = rtp.RequestsTransport.configure(settings)
```

Comment:
Next step for this test is to create a fake "requests" module (perhaps a pytest fixture) and then instantiate `RequestsTransport` directly passing in the fake:

```
test = rtp.RequestsTransport(settings, fake_requests)
```